core/ui/PageTemplate/story.tid

```diff
@@ -14,7 +14,11 @@ tags: $:/tags/PageTemplate
 
 </section>
 
-<$list filter="[list[$:/StoryList]]" history="$:/HistoryList" template="$:/core/ui/StoryTiddlerTemplate" storyview={{$:/view}} emptyMessage={{$:/config/EmptyStoryMessage}}/>
+<$vars searchPrefix="[" searchSuffix="]">
```

Comment:
we use `$let` instead of vars now. But as I wrote those helpers should be defined with the function

Reply:
Thanks